.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,38 @@
+name: pypi publishing
+
+
+on:
+  push:
+    tags: 'v[0-9]+.[0-9]+.[0-9]+'
+    branches: master
+
+
+jobs:
+
+  publish_on_pypi:
+
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: checkout latest commit
+        uses: actions/checkout@v2
+
+      - name: setup python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: create source distribution
+        run: |
+            python setup.py sdist
+
+      - name: create wheel distribution
+        run: |
+          python setup.py sdist
+
+      - name: publish package to pypi
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_PASSWORD }}
```

Comment:
This secret is repository specific, right? Note that I do not have sufficient rights (on this github repository) and I cannot add any secrets. So I guess it will always be pushed via your PyPI account and you already defined `secret.PYPI_PASSWORD` with you PyPI token, right?

Reply:
> This secret is repository specific, right?

Yes, I created a project-specific token on PyPI that is only valid for pushing to the `atlalign` project.


> I cannot add any secrets.

I tried to follow what was written [here](https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/#saving-credentials-on-github) to add my token to this GitHub repo secrets.

Reply:
:) Nice. I guess it is not necessary for me to have access to secrets on this repo since you already set everything up!